packages/firebase_ml_vision/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.0
+
+* **Breaking Change** Update TextDetector to TextRecognizer for android mlkit '17.0.0' and
+firebase-ios-sdk '5.6.0'.
```

Comment:
This is done in PRs: https://github.com/flutter/plugins/pull/733 and https://github.com/flutter/plugins/pull/732